src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -141,7 +144,20 @@ export default class MultipleProjectSelector extends React.PureComponent {
     } = this.props;
     const selectedProjectIds = new Set(value);
 
-    const allProjects = [...projects, ...nonMemberProjects];
+    const metaOptions = [];
+    if (multi) {
+      metaOptions.unshift({
+        id: null,
+        slug: t('My Projects'),
+      });
+    }
+    if (multi && organization.features.includes('open-membership')) {
+      metaOptions.unshift({
+        id: -1,
```

Comment:
Might be better to use a named constant here. It can be imported and reused at `app/views/releases/list/index.jsx` Line 125 and will make that bit of code more obvious.